app/templates/error.html

```diff
@@ -1,5 +1,9 @@
-<div class="alert alert-danger" role="alert" ng-show="message">
+<div class="alert alert-danger" role="alert" ng-show="errors.length">
   <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
-  <span class="sr-only">Error:</span>
-  {{ message }}
+  <span class="sr-only">Errors:</span>
+  <ul class="error-list">
```

Comment:
Er zit al een ng-show bovenin die de hele error niet toont, maar die is beter met een ng-if idd.